platforms-serverless/gcp-functions/run.sh

```diff
@@ -10,7 +10,7 @@ if [[ -z "${PRISMA_FORCE_NAPI+x}" ]]; then
   # use the default schema at prisma/schema.prisma file
   true
 else
-  mv ./prisma/schema-with-napi.prisma ./prisma/schema.prisma
```

Comment:
what triggered this? just curious - moving would make sure the other file is gone.

Reply:
The retry logic then fails as it can't move a file thats already been moved

Reply:
Ah, commit message: `@williamluke4 change mv to cp to prevent retries from failing`
And this is of course the case because we retry multiple times with the same checkout.
But doesn't this also lead to the binary test to be a library test the next run? (So we actually would need to backup the file, and write it back after the test?)

Reply:
No, because the binary and Node-API tests are in different jobs (i.e clean)

Reply:
Ah true, we changed that.